inc/saiqueue.h

```diff
@@ -433,6 +433,12 @@ typedef enum _sai_queue_stat_t
     /** Get watermark queue shared occupancy in cells [uint64_t] */
     SAI_QUEUE_STAT_SHARED_WATERMARK_CELLS = 0x0000002c,
 
+    /** Packets trimmed but failed to be admitted on a new queue (for trimmed packets) due to congestion. Counted on the original trimming-eligible queue [uint64_t] */
+    SAI_QUEUE_STAT_TRIM_DROPPED_PACKETS = 0x0000002d,
+
+    /** Packets trimmed and successfully transmitted on a new queue (for trimmed packets). Counted on the original trimming-eligible queue [uint64_t] */
+    SAI_QUEUE_STAT_TRIM_SENT_PACKETS = 0x0000002e,
```

Comment:
Please change it to
SAI_QUEUE_STAT_TX_TRIM_PACKETS

Reply:
done